Products/Jobber/tests/test_logformattercontext.py

```diff
@@ -0,0 +1,48 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2019, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+from __future__ import absolute_import, print_function
```

Comment:
I don't see we use print anywhere, do we need print_function ?

Reply:
The print function was used when I was debugging the code.

It's OK to leave the `print_function` reference in place.  It's not actually importing anything since __future__ is a built-in.  My linter (flake8) is not complaining.